depthimage_to_laserscan:
  ros__parameters:
    scan_time: 0.1
    range_min: 0.2
    range_max: 10.0
    scan_height: 1
    output_frame: "front_cam_link"
    use_sim_time: false
